bears/rest/reSTLintBear.py

```diff
@@ -14,13 +28,17 @@ class reSTLintBear(LocalBear):
     LICENSE = 'AGPL-3.0'
     CAN_DETECT = {'Formatting', 'Syntax'}
 
-    def run(self, filename, file):
+    def run(self, filename, file, ignore_unknown_roles: bool=False):
```

Comment:
I am not really sure, there is no feature "unimplemented" in restlint because both uses docutils

However, I feel restlint is the "purer" form of a rst linter, because rstcheck has "batteries" included like sphinx support etc. This is the only argument I have,

That said, I am up for deprecating `restlint` for the sake of bear uniformity.